src/qml/README.md

```diff
@@ -71,3 +71,9 @@ you must configure with the following option:
 ```
 ./configure --with-qml
 ```
+### Run
+
+To run the GUI QML application execute the file:
```

Comment:
2. To maintain consistency with line no. 68
`To add support for building the qml GUI,`
It would be best if you made the following change to the added message:
```suggestion
To run the qml GUI, execute the file:
```
It is better to keep terminology in our docs consistent, to begin with.

3. This one is a non-essential suggestion. How about mentioning, in the comment message itself, that the command should be run from root directory:
```suggestion
To run the qml GUI, run the following command from the root directory.
```